tracim/tracim/config/app_cfg.py

```diff
@@ -392,6 +392,9 @@ def __init__(self):
             'email.reply.use_txt_parsing',
             True,
         ))
+        self.EMAIL_REPLY_FILELOCK_PATH = tg.config.get(
+            'email.reply.filelock_path',
+        )
```

Comment:
default value: `"%(here)s/email_fetcher.lock"` ;)

Reply:
good point, but probably not in this location

Reply:
Yes after  face to face discuss inkhey suggest same thing.

Reply:
@inkhey Take care with  ̀'/tmp/tracim/email_fetcher.lock'` this default value: Be sure where it is used to create folders in the path. When the default value contain a possible non created folder, we should take care of it.